kofam_scan/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM quay.io/microbiome-informatics/miniconda:4.11.0
+
+ENV VERSION="1.3.0"
+
+LABEL maintainer="Microbiome Informatics Team www.ebi.ac.uk/metagenomics"
+LABEL base_image="ubuntu:20.04"
```

Comment:
This one is "microbiome-informatics/miniconda:4.11.0" not ubuntu, we need to fix in other Dockerfiles too.